cypress/integration/host/host_events.ts

```diff
@@ -12,6 +12,8 @@ describe("Host events", () => {
 
   it("host events display the correct text", () => {
     cy.visit(pathWithEvents);
+    cy.dataTestId("host-event-table-page-size-selector").click();
+    cy.dataTestId("host-event-table-page-size-selector-100").click();
```

Comment:
this test asserts that pagination component can be clicked but it does not assert that the correct number of hosts appear after pagination is selected